src/pages/Task.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from "react";
+import { RouteComponentProps } from "react-router-dom";
+import { useParams, useHistory } from "react-router-dom";
+
+const DEFAULT_TAB = "logs";
```

Comment:
this would be a good place to use TS enums or just a TS type. Something like:

```
enum Tabs {
  TESTS = tests
  LOGS = logs
  FILES = files
  BUILD_BARON = build-baron
}

const DEFAULT_TAB = Tabs.LOGS
```

Or...

```
type Tab = "logs" | "tests" | "files"
```
This way the value for tab can only ever be set to these values.